pkg/receive/handler.go

```diff
@@ -49,6 +56,8 @@ func NewHandler(logger log.Logger, forwardURL string, reg prometheus.Registerer)
 		reg.MustRegister(h.forwardRequestsTotal)
 	}
 
+	h.logger.Log("msg", "running receive handler")
```

Comment:
this log is a bit funny to me sine we're not running a goroutine here but creating a struct

Reply:
this might have been a test of mine with the logger not logging anything at first... :grin: